### ISSUE TITLE
Dropdown : Screen reader does not announce the error message is displaying after activating the 'Show error message' control 

### ISSUE THREAD

**Issue by 999-Dev** [OfficeDev/office-ui-fabric-react#10931]
**User Experience:**
Screen reader users will be confused if screen reader will not read the error message which is appeared after activating the 'Show error message' control. 

**Test Environment:**
OS: Version: RS5_Prerelease(1903 OS Build 19005.1)
URL: https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown
Edge Browser Version:
Microsoft Edge: 44.19005.1.0
Microsoft EdgeHTML: 18.19005
Chrome Browser Version:
Chrome: 77.0.3865.120

**Usages:**
Dropdown with error message

**Repro Steps:**
1.https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown in Edge and turn on Narrator.
2.Navigate to 'Basic input' button and activate it.
3.Navigate to 'Dropdown' link and activate it.
4.'Dropdown' pane will be appeared.
5.Navigate to 'Show error message' control and activate it.
6.Error message will be appeared and than verify the issue.

**Actual Result:**
Screen reader does not announce the error message after activating the 'show error message' control. Screen reader does not announce "This dropdown has an error".

**Expected Result:**
Screen reader should read the error message "This dropdown has an error" after activating the control.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=3ff80ac8-a17c-49c5-893b-34a0390e615b

**Attachment:**
![image](https://user-images.githubusercontent.com/55537167/67264901-053d8600-f4ca-11e9-96e9-03e9527673f0.png)
[OneDrive_2019-10-22.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3754201/OneDrive_2019-10-22.zip)

**Comment by jdhuntington** [comment]
Thanks, I believe this needs an alert.
